index.js

```diff
@@ -236,6 +237,101 @@ Peer.prototype.send = function (chunk) {
   self._channel.send(chunk)
 }
 
+/**
+ * Add a MediaStream to the connection.
+ * @param {MediaStream} stream
+ */
+Peer.prototype.addStream = function (stream) {
+  var self = this
+
+  self._debug('addStream()')
+
+  stream.getTracks().forEach(function (track) {
+    self.addTrack(track, stream)
+  })
+}
+
+/**
+ * Add a MediaStreamTrack to the connection.
+ * @param {MediaStreamTrack} track
+ */
+Peer.prototype.addTrack = function (track, stream) {
+  var self = this
+
+  self._debug('addTrack()')
+
+  var sender = self._pc.addTrack(track, stream)
+  self._senderMap.set(track, sender)
```

Comment:
`_senderMap` should be a `WeakMap` of `WeakMap`s and track should be added/removed to/from specific stream, since the same track theoretically might be also used for another stream. Otherwise it will override previous `sender` with new `sender`, which corresponds to different stream and previous one will be lost.